packages/ai-gemini/tests/gemini-adapter.test.ts

```
@@ -1,6 +1,6 @@
 import { describe, it, expect, beforeEach, vi } from 'vitest'
 import { z } from 'zod'
-import { chat, summarize } from '@tanstack/ai'
+import { chat, EventType, summarize } from '@tanstack/ai'
```

ESLint

[error] Member 'EventType' of the import declaration should be sorted alphabetically.

(sort-imports)